### PR TITLE
[civ2][lint/4] migrate easy lint rules over to civ02

### PR DESCRIPTION
**Pull request by can-anyscale**
Migrate more lints to civ02. Lint that remain in civ01 is a bit harder to migrate due to their dependencies.

Test:
- CI

### CODE REVIEW

File: python/requirements/lint-requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 clang-format==12.0.1
+docutils
```
Comment (aslonnie) [inline python/requirements/lint-requirements.txt:2]: should we pin the version?

Comment (can-anyscale) [reply python/requirements/lint-requirements.txt:2]: It's pinned in the compiled version. These non-compiled requirements should have not pin anything actually.

**Comment by can-anyscale** [comment]
oh, the ci/cd jobs are failing

**Comment by aslonnie** [comment]
> oh, the ci/cd jobs are failing; worked on retries

seems like github artifacts downloading were failing.

I should probably configure it to retry at least once for all cases on civ2 pipelines.

File: ci/lint/check-dashboard-format.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Checks front-end code format
+
+set -euxo pipefail
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE:-$0}")" || exit; pwd)/.."
+WORKSPACE_DIR="${ROOT_DIR}/.."
+
+cd "${WORKSPACE_DIR}"/python/ray/dashboard/client || exit
+
+if [ -z "${BUILDKITE-}" ]; then
```
Comment (aslonnie) [inline ci/lint/check-dashboard-format.sh:11]: double bracket? `if [[ -z "${BUILDKITE-}" ]]; then`

we can also probably just install npm inside the forge, and remove this.

**Review by aslonnie — APPROVED** [review]

**Pull request merged**